### PR TITLE
Improved display of movie's duration

### PR DESCRIPTION
**Pull request by jeremyhalin**
127 min is now " 2 hours 7 minutes "

### CODE REVIEW

File: language/fr.json [hunk 1]
```diff
@@ -1,62 +1,73 @@
...
+	"romanian": "Roumain",
+	"turkish": "Turc",
+	"unitH": {
+		"one": "unitH",
+		"other": "unitHs"
+	},
+	"unitM": {
+		"one": "unitM",
+		"other": "unitMs"
+	},
```
Comment (jeromelefeuvre) [inline language/fr.json]: seems this is wrong, remove line 63-70

**Comment by jeromelefeuvre** [comment]
after that, will be :+1:  !!

**Comment by jeremyhalin** [comment]
There are some things I don't understand on Github :smile:

**Comment by jeromelefeuvre** [comment]
good refactor.

Last question, do you think will be better to display `1h 27min` or `1 hour 27 minutes` ?
Personnally, short is better (and take less spaces), WDYT ?

**Comment by sirmackan** [comment]
Short version looks better :)

**Comment by jeremyhalin** [comment]
I prefer the second one :disappointed: 
Anyway, I can change it, it's not important. 
So let's take `1h 27min`

**Comment by jeremyhalin** [comment]
`1h 27min` or `1 h 27 min` ?

**Comment by jeromelefeuvre** [comment]
`1h 27min`

**Comment by jeremyhalin** [comment]
Done :white_check_mark:
But we will need to add new languages...

**Comment by jeremyhalin** [comment]
I added all new languages. Is something missing ?

**Comment by jeromelefeuvre** [comment]
seems good except fr.json. Indentation is fucked ;(

**Comment by EvandroZanatta** [comment]
+1

**Comment by EvandroZanatta** [comment]
what data should be inserted into JSON. I'll update the version pt-br

**Comment by jeremyhalin** [comment]
It's not commited yet so I don't know if you have to do that.
Otherwise, you can add
`"durationUnitM" : "[TRANSLATED]",
"durationUnitH" : "[TRANSLATED]",`

**Comment by davidgumberg** [comment]
Hey @HJerem! Thanks for the contribution! Unfortunately this project is no longer being maintained, but it is being forkd by the very reputable folks at YiFy, so please resubmit your pull request to: https://github.com/Yify/popcorn-app

**Pull request closed without merging**